lib/wallet/src/Cardano/Api/Gen.hs

```diff
@@ -1013,13 +1016,16 @@ genEpochNo :: Gen EpochNo
 genEpochNo = EpochNo <$> arbitrary
 
 genCostModel :: Gen CostModel
-genCostModel = case Plutus.defaultCostModelParams of
-  Nothing -> error "Plutus defaultCostModelParams is broken."
-  Just dcm ->
-      CostModel
-    -- TODO This needs to be the cost model struct for whichever
-    -- Plutus version we're using, once we support multiple Plutus versions.
-    <$> mapM (const $ chooseInteger (0, 5_000)) dcm
+genCostModel = do
+  let costModelParams = Alonzo.getCostModelParams Plutus.testingCostModelV1
+  eCostModel <- Alonzo.mkCostModel <$> genPlutusLanguage
+                                   <*> mapM (const $ chooseInteger (0, 5_000)) costModelParams
+  case eCostModel of
+    Left err -> error $ "genCostModel: " ++ show err
+    Right cModel -> return . CostModel $ Alonzo.getCostModelParams cModel
```

Comment:
Just some minor formatting fixes (our house style favours newline + 4-space-indent instead of vertical alignment).

```suggestion
    let costModelParams = Alonzo.getCostModelParams Plutus.testingCostModelV1
    eCostModel <- Alonzo.mkCostModel
        <$> genPlutusLanguage
        <*> mapM (const $ chooseInteger (0, 5_000)) costModelParams
    case eCostModel of
        Left err -> error $ "genCostModel: " ++ show err
        Right cModel -> return . CostModel $ Alonzo.getCostModelParams cModel
```